spec/classes/config_spec.rb

```diff
@@ -13,6 +13,9 @@
       end
 
       describe "rundeck::config class without any parameters on #{os}" do
+
```

Comment:
Even though these are tested as private classes, for some reason, this test still worked even before I moved it....